modules/local/starfusion/build/main.nf

```diff
@@ -1,37 +1,32 @@
 process STARFUSION_BUILD {
-    tag 'star-fusion'
+    tag "$meta.id"
+    label 'process_high'
 
     conda "${moduleDir}/environment.yml"
-    container 'community.wave.seqera.io/library/dfam_hmmer_samtools_star-fusion_pruned:5694d82381bf039e'
+    container 'community.wave.seqera.io/library/dfam_hmmer_minimap2_samtools_pruned:63e3d21ca68ea531'
 
     input:
     tuple val(meta), path(fasta)
     tuple val(meta2), path(gtf)
+    path fusion_annot_lib
+    val dfam_species
 
     output:
-    path "*"  , emit: reference
+    path "ctat_genome_lib_build_dir"  , emit: reference
 
     script:
-    def binPath = (workflow.profile.tokenize(',').intersect(['conda', 'mamba']).size() >= 1)  ? "prep_genome_lib.pl" : "/usr/local/src/STAR-Fusion/ctat-genome-lib-builder/prep_genome_lib.pl"
+    if (dfam_species != "human" && dfam_species != "mouse") {
+        error "Invalid species for --dfam_db. Only 'human' or 'mouse' are accepted. Provided: ${dfam_species}"
```

Comment:
As a principle I think rnafusion should supports human only, but should be flexible enough to allow tweaking for other species for people to do at their own risk, so I would remove that check

Reply:
This one is due to the tool. While it can accept files as input, when doing the automatic download it can only accept human or mouse: 

https://github.com/TrinityCTAT/ctat-genome-lib-builder/blob/1cf500c07a67c0240228b8f9c245c0db3e507e4f/prep_genome_lib.pl#L38